### ISSUE TITLE
Plugin manager using plugin class name, but it should use name member (Trac #91)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#95]
Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/91
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Plugin manager using plugin class name, but it should use name member", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-04-03T15:03:31", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120403T15:14:37: 
* ryan changed owner from "" to "cfobel"
* ryan changed status from "new" to "assigned"

**Comment by cfobel** [comment]
Trac update at 20120408T01:03:11: 
* christian@fobel.net commented:

> commit 355b31dd79d8d084eace81f24c18e4d9c2fe8fff
> Author: Christian Fobel <christian@fobel.net>
> Date:   Sat Apr 7 20:58:40 2012 -0400
> 
>     Fix #95: use plugin name member in plugin manager


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "assigned" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "" to "microdrop"
